client/trino-client/src/main/java/io/trino/client/StageStats.java

```diff
@@ -153,6 +159,12 @@ public long getPhysicalInputBytes()
         return physicalInputBytes;
     }
 
+    @JsonProperty
+    public long getPhysicalInputDataSize() { return physicalInputDataSize; }
```

Comment:
fix formatting

Reply:
> 

I have [AirLift code style](https://github.com/airlift/codestyle) configured for my intelliJ. This is the default formatting it applies when I reformat the file. Could you elaborate on what needs to be changed here ? Thanks!

Reply:
single-line getters and setters are not the existing convention, try to keep the code consistent with it's surroundings first and foremost.